android/src/test/java/com/rudderstack/android/RudderAnalyticsTest.kt

```diff
@@ -39,7 +39,7 @@ import org.robolectric.annotation.Config
 
     @Test
     fun `test put anonymous id`() {
```

Comment:
I know this is not part of the changes, but as we follow the boy scout rule "leave this better than you found" can you improve also this test name? Something like:
```
    @Test
    fun `given writeKey and configuration are passed, when anonymousId id is set, then assert that configuration has this anonymousId set as a property`() {
```
would be great!